### ISSUE TITLE
Support tags as an (additional) accessibility feature

### ISSUE THREAD

**Issue by SoniEx2** [tootsuite/mastodon#6705]
You should be able to use tags like `#cw` and `#me` (if we ever get action toots) and `#public`/`#followers`/`#unlisted`/`#dm` first thing in a toot as an accessibility feature, in addition to other accessibility features.

THIS MUST NOT REPLACE OTHER ACCESSIBILITY FEATURES.

It should be opt-out, tho, for those who wanna use those tags first thing in a toot for some reason. (But this is pretty unlikely, which is why opt-out is suitable. Also see below.)

For `#cw`, either use the current like (i.e. `#cw cw text goes here`) or use it as a separator (i.e. `cw text goes here #cw toot content goes here`). (See unresolved questions.)

To escape them, add a newline at the beginning. (You can't start a toot with a newline already, so this won't be an issue.)

Because they get stripped, you can also add a newline to escape the next. E.g.

```
#public #cw hello
how are you?
```

Would become

```
hello [show more]
how are you?
```

But
```
#public
#cw hello
how are you?
```

Would become

```
#cw hello
how are you?
```

Unresolved questions:

- [ ] Should `#cw` come before or after cw text?
- [ ] Should we keep parsing these after a `#cw` line? E.g. should
    ```
    #cw hello
    #public test
    ```
    parse as a public toot with "hello" cw and "test" contents?

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).
